web/src/lib/components/shared-components/search-bar/search-bar.svelte

```diff
@@ -84,7 +85,18 @@
   };
 </script>
 
-<div class="w-full relative" use:clickOutside on:outclick={onFocusOut} on:escape={onFocusOut}>
+<div
+  class="w-full relative"
+  use:clickOutside
+  on:outclick={onFocusOut}
+  tabindex="-1"
+  use:shortcut={{
+    shortcut: { key: 'Escape' },
+    onShortcut: () => {
+      onFocusOut();
```

Comment:
This doesn't work if you clear focus by clicking on an empty area in the search bar. Moving the shortcut to `window` should solve that

Reply:
Pushed out a fix - I noticed that the search field also needed an "Escape" key handler, to close the recent searches dropdown.